app/classifier/classifier.jsx

```diff
@@ -239,7 +246,15 @@ class Classifier extends React.Component {
           onChange={this.handleAnnotationChange.bind(this, currentClassification)}
           playIterations={this.props.workflow.configuration.playIterations}
         />
-
+        {this.props.workflow.configuration.metadata &&
```

Comment:
We've re-written the classifier to get rid of project-specific, or workflow-specific, hacks. This conditional makes me nervous that those are creeping back in again.

Reply:
Good point, how would you suggest I phrased this so that the model renderer is only loaded for modelling workflows?

Reply:
Maybe incorporate the `ModelCanvas` into the subject viewer as a new type of frame viewer?

Reply:
That would remove the need to check the workflow at all, since the model would become just another frame in the subject locations array.